In [1]:
import math as m
import numpy as np

### Задача 1

 Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

[Xср - Zα/2 * sigma/sqrt(n); Xср + Zα/2 * sigma/sqrt(n)]

In [9]:
80 - 1.96 * 16 / m.sqrt(256)

78.04

In [10]:
80 + 1.96 * 16 / m.sqrt(256)

81.96

### Задача 2

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

[Xср - tα/2 * sigma/sqrt(n); Xср + tα/2 * sigma/sqrt(n)]

In [21]:
x = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]

In [22]:
Xср = sum(x) / len(x)
Xср

6.589999999999999

In [23]:
sigma = np.std(x, ddof=1)
sigma

0.4508017549014448

In [24]:
Xср - 2.262 * sigma / m.sqrt(10)

6.267538255912425

In [25]:
Xср + 2.262 * sigma / m.sqrt(10)

6.912461744087573

[6.267538255912425; 6.912461744087573]

### Задача 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [20]:
# H0, диаметр = 17 мм., mu = mu0
# H1, диаметр = 17,5 мм., mu > mu0

In [18]:
Zт = 1,645

In [17]:
Zн = (17.5 - 17) / (m.sqrt(4) / m.sqrt(100))
Zн

2.5

Zт < Zн, верна гипотеза Н1 на уровне знчимости α=0,05

### Задача 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [27]:
# H0, mu = mu0
# H1, mu не равно mu0

In [26]:
x = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]

In [28]:
sigma = np.std(x, ddof=1)
sigma

4.453463071962462

In [29]:
Xср = sum(x) / len(x)
Xср

198.5

In [33]:
tн1 = (Xср - 200) / (sigma / 10)
tн1

-3.368165348543039

In [34]:
tн2 = (Xср + 200) / (sigma / 10)
tн2

894.8092609296008

In [35]:
tт = 3,250

tн1 < tт < tн2, гипотеза Н0 верна, на уровне значимости 0,01